code/analytics/src/main/java/com/adobe/marketing/mobile/analytics/internal/AnalyticsDatabase.kt

```diff
@@ -19,6 +19,40 @@ import com.adobe.marketing.mobile.services.Log
 import com.adobe.marketing.mobile.services.PersistentHitQueue
 import com.adobe.marketing.mobile.services.ServiceProvider
 
+/**
+ * The Analytics hit database which queues and persists hits before processing.
+ *
+ * Analytics hit reordering
+ * If backDateSessionInfo and offlineTracking is enabled, we should send hit with previous session
+ * information / crash information before sending any hits for current session.
+ * (We get this information from `lifecycle.responseContent` event)
```

Comment:
In general "we" should use impersonal comments, can you reword the comments to not use "we"?